### DISCUSSION TITLE
Fun if there will be Polish language

### DISCUSSION THREAD

**Discussion by retrogamer140996** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#429] [Ideas]
Fun if there will be polish language added in

**Comment by sumeshir26** [comment]
Hello @retrogamer140996, 
Even if we tried, we cannot add that language because it is not supported by the Google Assistant SDK. You can find the supported languages [here](https://developers.google.com/assistant/sdk/reference/rpc/languages).
